client/components/count/index.jsx

```diff
@@ -30,6 +27,7 @@ Count.propTypes = {
 Count.defaultProps = {
 	primary: false,
 	compact: false,
+	numberFormat: ( i ) => i,
```

Comment:
🤔 Where does the `numberFormat` come from, now that we removed the `localize` HoC?

Reply:
Oops, you're right. I wasn't aware `numberFormat` is coming from `i18n-calypso`. Fixed via https://github.com/Automattic/wp-calypso/pull/64028/commits/b3d4055e199668d2c552784dd99ca23ae2d90337